packages/one-app-bundler/webpack/module/webpack.client.js

```diff
@@ -55,6 +55,8 @@ const webpackClient = async (babelEnv) => {
           fs: false,
           module: false,
           net: false,
+          util: false,
```

Comment:
Im confused. The docs on webpack say that providing `false` here will NOT include the polyfill.
https://webpack.js.org/configuration/resolve/#resolvefallback

> bundler must provide polyfils for node:utils and node:url

But in the description of the PR, it seems like your intention is to include a polyfil?